core/src/main/java/dev/triumphteam/gui/guis/BaseGui.java

```diff
@@ -186,6 +193,15 @@ public BaseGui(@NotNull final GuiType guiType, @NotNull final String title) {
         guiItems = new LinkedHashMap<>();
     }
 
+    private boolean checkFolia() {
```

Comment:
Should probably move this method to https://github.com/TriumphTeam/triumph-gui/blob/master/core/src/main/java/dev/triumphteam/gui/components/util/VersionHelper.java since it's what handles most reflection/version/server-type things.